.github/workflows/run_tests.yml

```diff
@@ -14,6 +14,9 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10]
```

Comment:
It wouldn't change python version as the tests here runs in docker and are not affected by it. So we need to duplicate poetry install + run tests here. Let's think about it later